common/device-model.cpp

```diff
@@ -3376,93 +3395,126 @@ namespace rs2
                 }
                 if (ImGui::IsItemHovered())
                 {
-                    std::string tooltip = rsutils::string::from()
-                        << "On-Chip Calibration"
-                        << (streaming ? " (Disabled while streaming)" : "");
+                    std::string tooltip;
+                    if( is_d555 )
+                        tooltip = "This will improve the depth noise.\n"
+                                  "Point at a scene that normally would have > 50 %% valid depth pixels,\n"
+                                  "then press calibrate."
+                                  "The health-check will be calculated.\n"
+                                  "If >0.25 we recommend applying the new calibration.\n"
```

Comment:
`if the result is bigger than 0.25`?

Reply:
I think the score relates to a ratio that is minimized by the calibration algorithm. Under 0.25 is good. 0.25 to 0.75 is poor. Over 0.75 is bad.
Again, this was copied as is from D400.